lua/nvim-highlight-colors/color/utils.lua

```diff
@@ -116,6 +121,21 @@ function M.get_tailwind_named_color_value(color)
 	end
 end
 
+---Returns the hex value of a python ansi color
+---@param color string
+---@return string|nil
+---@usage get_ansi_named_color_value("\033[31m") => Returns '#FF0000'
+function M.get_ansi_named_color_value(color)
+	local color_code = nil
+	if string.match(color, "\\033[[0-9;]*m") then
```

Comment:
Maybe you could reuse `M.ansi_regex = "\\033%[[0-9;]*m"` here

Reply:
Hey is this what you were meaning? 

```
function M.get_ansi_named_color_value(color)
	local color_code = nil
	if string.match(color, patterns.ansi_regex) then
```